HerPublicWebsite.BusinessLogic/Models/ReferralRequest.cs

```diff
@@ -60,7 +61,10 @@ public ReferralRequest(Questionnaire questionnaire)
         FullName = questionnaire.LaContactName;
         ContactEmailAddress = questionnaire.LaContactEmailAddress;
         ContactTelephone = questionnaire.LaContactTelephone;
+        
         RequestDate = DateTime.Now;
+        WasSubmittedToPendingLocalAuthority =
+            questionnaire.LocalAuthorityHug2Status == LocalAuthorityData.Hug2Status.Pending;
```

Comment:
Just checking if you saw this comment: https://github.com/UKGovernmentBEIS/desnz-home-energy-retrofit-beta/pull/274#issuecomment-1973594981

> Please could you also add a test for this in ReferralRequestTests?

Could you please add a test for this flag being initiated property?

Reply:
That's in progress now :) I just pushed the non-test changes first since the test felt more self-contained